File: android/filtering-content/GetItemsByTaxonomyTerm.java
// DocSection: filtering_get_items_by_taxonomy_term
// Note: Filters work with codenames of the tags.
// Gets items tagged with one specific tag
List<ContentItem> items = deliveryService.<ContentItem>items()
    .containsFilter("elements.tags", Arrays.asList("kentico"))
    .get()
    .getItems();

// Gets items tagged with a list of specific tags
List<object> items = deliveryService.<object>items()
    .allFilter("elements.tags", Arrays.asList("kontent", "headless"))
    .get()
    .getItems();

// Gets items tagged with at least one of multiple tags
List<object> items = deliveryService.<object>items()
    .anyFilter("elements.tags", Arrays.asList("football", "soccer"))
    .get()
    .getItems();
// EndDocSection

File: net/filtering-content/GetItemsByTaxonomyTerm.cs
// DocSection: filtering_get_items_by_taxonomy_term
DeliveryItemListingResponse<object> response = await _deliveryClient.GetItemsAsync<object>(
  // Get articles tagged with specific tag
  new ContainsFilter("elements.tags", "kentico")
);

DeliveryItemListingResponse<object> response = await _deliveryClient.GetItemsAsync<object>(
  //Get articles tagged with any of the below tags
  new AnyFilter("elements.tags", "sport", "soccer")
);

DeliveryItemListingResponse<object> response = await _deliveryClient.GetItemsAsync<object>(
  //Get articles tagged with all of the below tags
  new AllFilter("elements.tags", "mvc", "kontent", "headless")
);
// EndDocSection
